crates/holochain/src/core/workflow/app_validation_workflow/validation_dependencies.rs

```diff
@@ -27,7 +27,7 @@ impl Default for ValidationDependencies {
 }
 
 impl ValidationDependencies {
-    const FETCH_TIMEOUT: Duration = Duration::from_secs(60);
+    const FETCH_TIMEOUT: Duration = Duration::from_secs(30);
```

Comment:
Why?

Reply:
My reasoning is that if there are a few hashes that cannot be fetched, app validation will retrigger every couple of 100 ms. It would continue for 60 seconds, so I thought 30 seconds is a compromise between a frequent re-trigger and enough time to fetch hashes from a slow node.

Reply:
This seems inconsistent given that we allow 60 sec for fetching data in other contexts. It seems like if we are willing to reduce the timeout here, we should reduce it everywhere, because similar reasoning applies in other situations, especially because it's been asserted that running validation is not very expensive, so a constant factor of 2 in worst-case time difference should not have that much effect compared to the actual effect of waiting 60 seconds to determine the fetchability of some data.

Reply:
Fine, agreed.